rhoas-cli/README.adoc

```diff
@@ -72,68 +72,44 @@ NOTE: The `rhoas` link:https://github.com/redhat-developer/app-services-cli/rele
 [discrete,id="installing-rhoas-cli-linux_{context}"]
 === Installing the rhoas CLI on Linux
 
-If you are using Linux, you can install `rhoas` by either using the installation script, or the RPM.
-The installation script installs the latest version of `rhoas` by default.
+.Procedure
 
-[discrete,id="installation-script_{context}"]
-==== Installation script
+. Navigate to the https://developers.redhat.com/products/red-hat-openshift-streams-for-apache-kafka/overview[Red Hat OpenShift Streams for Apache Kafka] page on the Red Hat Developer Portal.
 
-.Procedure
+. Click https://developers.redhat.com/products/red-hat-openshift-streams-for-apache-kafka/download[*Install/Specifics*] and navigate to the bottom of the page to the *All Downloads* section.
```

Comment:
```suggestion
. Click https://developers.redhat.com/products/red-hat-openshift-streams-for-apache-kafka/download[*Install/specifics*] and scroll to the bottom of the page to the *All Downloads* section.
```

Reply:
The button title on the website is *Install/specifics* so I think we should keep it the same.

Reply:
I just meant that the tab is "Install/specifics" with a lowercase "s" (as opposed to "Install/Specifics").

Reply:
Sorry, yes. I got the suggestion the wrong way around in my head :)